packages/react-components/react-skeleton/src/components/SkeletonItem/useSkeletonItemStyles.ts

```diff
@@ -0,0 +1,195 @@
+        Window 50%,
+        transparent 100%)
+      `,
+    },
+  },
+  waveRtl: {
+    animationName: skeletonWaveAnimationRTL,
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralStencil1} 0%,
+      ${tokens.colorNeutralStencil2} 50%,
+      ${tokens.colorNeutralStencil1} 100%)`,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: `WindowText
+      linear-gradient(
+        to right,
+        transparent 0%,
+        Window 50%,
+        transparent 100%)
+      `,
+    },
+  },
+  pulse: {
+    animationName: skeletonPulseAnimation,
+    backgroundColor: tokens.colorNeutralStencil1,
+  },
+  translucent: {
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralStencil1Alpha} 0%,
+      ${tokens.colorNeutralStencil2Alpha} 50%,
+      ${tokens.colorNeutralStencil1Alpha} 100%)`,
+  },
+  translucentPulse: {
+    backgroundColor: tokens.colorNeutralStencil1Alpha,
+  },
+});
+
+const useRectangleStyles = makeStyles({
+  root: {
+    width: 'auto',
+  },
+  8: { height: '8px' },
+  12: { height: '12px' },
+  16: { height: '16px' },
+  20: { height: '20px' },
+  24: { height: '24px' },
+  28: { height: '28px' },
+  32: { height: '32px' },
+  36: { height: '36px' },
+  40: { height: '40px' },
+  48: { height: '48px' },
+  56: { height: '56px' },
+  64: { height: '64px' },
+  72: { height: '72px' },
+  96: { height: '96px' },
+  120: { height: '120px' },
+  128: { height: '128px' },
+});
+
+const useSquareSizeStyles = makeStyles({
+  8: { width: '8px', height: '8px' },
+  12: { width: '12px', height: '12px' },
+  16: { width: '16px', height: '16px' },
+  20: { width: '20px', height: '20px' },
+  24: { width: '24px', height: '24px' },
+  28: { width: '28px', height: '28px' },
+  32: { width: '32px', height: '32px' },
+  36: { width: '36px', height: '36px' },
+  40: { width: '40px', height: '40px' },
+  48: { width: '48px', height: '48px' },
+  56: { width: '56px', height: '56px' },
+  64: { width: '64px', height: '64px' },
+  72: { width: '72px', height: '72px' },
+  96: { width: '96px', height: '96px' },
+  120: { width: '120px', height: '120px' },
+  128: { width: '128px', height: '128px' },
+});
+
+const useCircleSizeStyles = makeStyles({
+  root: {
+    ...shorthands.borderRadius('50%'),
+  },
+  8: { width: '8px', height: '8px' },
+  12: { width: '12px', height: '12px' },
+  16: { width: '16px', height: '16px' },
+  20: { width: '20px', height: '20px' },
+  24: { width: '24px', height: '24px' },
+  28: { width: '28px', height: '28px' },
+  32: { width: '32px', height: '32px' },
+  36: { width: '36px', height: '36px' },
+  40: { width: '40px', height: '40px' },
+  48: { width: '48px', height: '48px' },
+  56: { width: '56px', height: '56px' },
+  64: { width: '64px', height: '64px' },
+  72: { width: '72px', height: '72px' },
+  96: { width: '96px', height: '96px' },
+  120: { width: '120px', height: '120px' },
+  128: { width: '128px', height: '128px' },
```

Comment:
You don't need to repeat this. You could rename `useSquareSizeStyles` => `useSizeStyles`, and just use it for both square and circle.